shared/test/test_regexp.rb

```diff
@@ -0,0 +1,53 @@
+require_relative 'test_helper'
+
+require 'cdo/regexp'
+class RegexpUtilsTest < Minitest::Test
+  EXPECTED_DIGITS = '1234567890'
+  VALID_US_PHONE_NUMBERS = %w[
```

Comment:
Thanks Brian. Funny I didn't find that one. I chatted with @tanyaparker and we prefer the new regex that will accept phone numbers as just a string of digits. I'll update this test.